pantry-finder-api/config/application.rb

```diff
@@ -3,6 +3,7 @@
 Jets.application.configure do
   config.project_name = 'pantry-finder-api'
   config.mode = 'api'
+  config.cors = true
```

Comment:
This is fine but it looks like changes from your other pr got muddled into here. Not a big deal but might lead to some merge conflicts.